required/latex-lab/build.lua

```diff
@@ -64,9 +64,14 @@ checkconfigs =
  {"build","config-TU","config-OR",
   "config-mathtagging",
   "config-OR-luatex",
-  "config-sec-tagging",
-  "config-toc-tagging",
+  "config-sec",
+  "config-toc",
   "config-block-tagging",
+  "config-graphic",
+  "config-minipage",
+  "config-float",
+  "config-bib",
+  "config-LM-tagging",
```

Comment:
OK, I can do that too. (There are still some -tagging around in some of the testphase key names and in the testfiles, I can try to get rid of them too, but it could affect older example files, so I didn't try that yet).

Reply:
we can wait with all that until after the release, but I just noticed it next to the other change